src/sg/gov/acra/bizfiles/profiles/company/1.0/schema.json

```diff
@@ -0,0 +1,211 @@
+        },
+        "originalIncorporationDeregistrationName": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/entityName"
+        },
+        "registrationCountry": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/country"
+        },
+        "registrationDate": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+        "deregistrationRegisteredName": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/entityName"
+        },
+        "conditions": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/conditions"
+        },
+        "activities": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/activities"
+        },
+        "capitals": {
+            "type": "array",
+            "items": {
+                "$ref": "../../../common/definitions/1.0/schema.json#/definitions/capitals"
+            }
+        },
+        "treasuryShares": {
+            "type": "array",
+            "items": {
+                "$ref": "../../../common/definitions/1.0/schema.json#/definitions/capitals"
+            }
+        },
+        "address": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/address"
+        },
+        "changeOfAddressDate": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+        "lastAgmDate": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+        "lastArDate": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+        "lastArDateFye": {
+            "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"
+        },
+        "auditFirms": {
+            "type": "array",
+            "items": {
+                "$ref": "../../../common/definitions/1.0/schema.json#/definitions/auditFirms"
+            }
+        },
+        "charges": {
+            "type": "array",
+            "items": {
+                "$ref": "../../../common/definitions/1.0/schema.json#/definitions/charges"
+            }
+        },
+        "representatives": {
+            "type": "array",
+            "items": {
+                "allOf": [
+                    {
+                        "$ref": "../../../common/definitions/1.0/schema.json#/definitions/person"
+                    }
+                ],
+                "properties": {
+                    "appointmentDate": {
+                        "type": "string",
+                        "format": "date",
+                        "pattern": "(((19|20)([2468][048]|[13579][26]|0[48])|2000)[-]02[-]29|((19|20)[0-9]{2}[-](0[469]|11)[-](0[1-9]|[12][0-9]|30)|(19|20)[0-9]{2}[-](0[13578]|1[02])[-](0[1-9]|[12][0-9]|3[01])|(19|20)[0-9]{2}[-]02[-](0[1-9]|1[0-9]|2[0-8])))"
```

Comment:
mmmm how come we not reusing back to date definition ?
` "$ref": "../../../common/definitions/1.0/schema.json#/definitions/date"`

Reply:
Good find, must have missed that one, will update along with disqualificationDate

Reply:
```suggestion
                        $ref": "../../../common/definitions/1.0/schema.json#/definitions/date
```